astro/src/content/docs/lifecycle/migrate-users/bulk/stytch.mdx

```diff
@@ -0,0 +1,327 @@
+---
+title: Migration From Stytch
+description: How to migrate your users from Stytch to FusionAuth.
+section: lifecycle
+subcategory: migrate users
+tertcategory: bulk
+prerequisites: Docker, Node
+importCodeRoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-import-scripts/master
+javaCodeRoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-contrib/master
+---
+
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import InlineField from 'src/components/InlineField.astro';
+import Aside from '/src/components/Aside.astro';
+import {RemoteCode} from '@fusionauth/astro-components';
+import AdditionalSupport from 'src/content/docs/lifecycle/migrate-users/bulk/_additional-support.mdx';
+
+## Overview
+
+This article explains how to import users from Stytch into FusionAuth. It is a low-level, technical tutorial focusing on transferring password hashes, calling APIs, and preparing data. The article does not explain the high-level process of a migration strategy. This is explained in the [migration overview article](/docs/lifecycle/migrate-users/bulk/general). Before proceeding, please ensure you understand concepts like dual write, backfill, final synchronization, rolling migration, and cutover. The [Stytch documentation on migration strategies](https://stytch.com/docs/guides/migrations/migrating-user-data-statically) is also useful reading. Be aware of all laws regarding the protection and transfer of personal information in your country.
```

Comment:
Should these be part of the Prerequisites?  Where can I go to learn about these concepts if I am not familiar with them? No that I follow the Stytch doc link I see they are there, but should be make more clear. Also I think info is in regards to importing users to Stytch. I don't think we use those terms in migrating to FusionAuth. Do we need them here?